README.md

```diff
@@ -1,3 +1,74 @@
 # Appcues Expo Module
 
 > **This is a pre-release alpha package in preparation for v4 of the Appcues SDK. It is not necessary for integrating v3 of the Appcues SDK. Please refer to the [Appcues React Native Module](https://github.com/appcues/appcues-react-native-module).**
+
+Appcues Expo Module allows you to integrate Appcues push notifications into your Expo app alongside the [Appcues React Native Module](https://github.com/appcues/appcues-react-native-module).
+
+- [Appcues Expo Module](#appcues-expo-module)
+  - [🚀 Getting Started](#-getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [🎬 Examples](#-examples)
+  - [👷 Contributing](#-contributing)
+  - [📄 License](#-license)
+
+## 🚀 Getting Started
+
+## Prerequisites
+
+1. Ensure the [Appcues React Native Module](https://github.com/appcues/appcues-react-native-module) is installed in your app and the SDK is initialized:
+    ```js
+    import * as Appcues from '@appcues/react-native'
+
+    await Appcues.setup('APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID')
+    ```
+
+2. It is recommended to have [configured your Android and iOS push settings in Appcues Studio](https://docs.appcues.com/en_US/push-notifications/push-notification-settings) before configuring push notifications in your app to allow you quickly test your configuration end to end.
+
+3. Copy your Firebase configuration file into your project and set the path to the file `app.json` `android.googleServicesFile` ([doc](https://docs.expo.dev/versions/latest/config/app/#googleservicesfile-1)) property:
```

Comment:
just reads a little funny - maybe `set the path to the file in your app.json file, in the android.googleServicesFile property` ?

Reply:
Thanks, I'm pretty sure that's what I meant to say